chrome/content/zotero/elements/contextPane.js

```diff
@@ -181,11 +181,6 @@
 				_contextPaneSplitter.setAttribute('hidden', false);
 
 				_contextPane.setAttribute('collapsed', !(_contextPaneSplitter.getAttribute('state') != 'collapsed'));
-				// It seems that on heavy load (i.e. syncing) the line below doesn't set the correct value,
-				// therefore we repeat the same operation at the end of JS message queue
-				setTimeout(() => {
-					_contextPane.setAttribute('collapsed', !(_contextPaneSplitter.getAttribute('state') != 'collapsed'));
```

Comment:
I believe this line is causing all the weirdness of the context pane/item pane. After #4289, we always use the correct `tabType` to decide which item pane to display and thus no longer need this trick.